django_extensions/management/commands/sync_s3.py

```diff
@@ -388,7 +388,7 @@ def upload_s3(self, arg, dirname, names, dirs):
                         print("\tgzipped: %dk to %dk" % (file_size / 1024, len(filedata) / 1024))
             if self.do_expires:
                 # HTTP/1.0
-                headers['Expires'] = '%s GMT' % (email.Utils.formatdate(time.mktime((datetime.datetime.now() + datetime.timedelta(days=365 * 2)).timetuple())))
+                headers['Expires'] = '%s GMT' % (email.utils.formatdate(time.mktime((datetime.datetime.now() + datetime.timedelta(days=365 * 2)).timetuple())))
```

Comment:
Currently only `email` is imported. In order to work correctly on both versions it should be imported as `import email.utils`. Probably right now it works correctly because somewhere else in the code `email.utils` is imported and now `email` package has `utils` module as attribute attached to it, but I think that it is more safe if `email.utils` is imported as a whole.